src/components/rootcomponents/Sidebar.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { FaShoppingCart, FaRegHeart } from 'react-icons/fa';
+import { MdOutlineDashboardCustomize } from 'react-icons/md';
+import { IoBagHandleOutline, IoLogOutOutline } from 'react-icons/io5';
+import { CiSettings } from 'react-icons/ci';
+import { MdStarOutline, MdOutlineNotificationsActive } from 'react-icons/md';
+import { NavLink } from 'react-router-dom';
+import { FiMenu } from 'react-icons/fi';
+
+const links = [
+  {
+    id: 1,
+    name: 'Dashboard',
+    icon: <MdOutlineDashboardCustomize className="text-2xl" />,
+    link: '/',
+  },
+  {
+    id: 2,
+    name: 'My orders',
+    icon: <IoBagHandleOutline className="text-2xl" />,
+    link: '/orders',
+  },
+  {
+    id: 3,
+    name: 'Reviews',
+    icon: <MdStarOutline className="text-2xl" />,
+    link: '/reviews',
+  },
+  {
+    id: 4,
+    name: 'Notifications',
+    icon: <MdOutlineNotificationsActive className="text-2xl" />,
+    link: '/notifications',
+  },
+  {
+    id: 5,
+    name: 'Wishlist',
+    icon: <FaRegHeart className="text-2xl" />,
+    link: '/wishlist',
+  },
+];
```

Comment:
Put this in the data folder under source. create a file there with relevant name, export it and use it here